base/src/main/scala/co/uproot/abandon/Config.scala

```diff
@@ -53,11 +53,21 @@ object SettingsHelper {
   }
 
   def makeSettings(configFileName: String) = {
+    def handleInput(input: String, confPath: String): List[String] = {
+      val parentPath = Processor.mkParentDirPath(confPath)
+      if (input.startsWith("glob:"))
+        FileUtils.globListFiles(input, parentPath)
+      else if (input.startsWith("regex:"))
+        FileUtils.regexListFiles(input, parentPath)
+      else
+        List(Processor.mkRelativeFileName(input, confPath))
```

Comment:
Thanks for the PR. Semi-minor remakr: Please use braces around if-else blocks or any other blocks.

An example where braces would have made a lot of difference: [Apple's SSL bug](https://www.imperialviolet.org/2014/02/22/applebug.html)